app/ai/voice/agents/breeze_buddy/handlers/internal/end_conversation.py

```
@@ -75,6 +77,75 @@ async def end_conversation(context: TemplateContext, args, transition_to=None):
+                    )
+                if context.lead:
+                    context.root_span.set_attribute(
+                        "attempt_count", context.lead.attempt_count + 1
+                    )
+
+                # Extract system prompt from template config
+                try:
+                    system_prompt = _extract_system_prompt(context)
+                    if system_prompt:
+                        context.root_span.set_attribute("system_prompt", system_prompt)
+                except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)